config.yaml

```diff
@@ -15,6 +15,28 @@ options:
       to accept the charm default. Choosing 'auto' will install the latest 
       supported version of etcd at deploy time, but will not automatically upgrade
       to a newer version thereafter.
+  heartbeat_interval:
+    type: int
+    default: 100
+    description: |
+      The frequency with which the leader will notify followers that it is
+      still the leader. By default, etcd uses a 100ms heartbeat interval.
+  election_timeout:
+    type: int
+    default: 1000
+    description: |
+      How long a follower node will go without hearing a heartbeat before
+      attempting to become leader itself.
+      By default, etcd uses a 1000ms election timeout.
```

Comment:
this default matches the docs

> **--election-timeout**
    Time (in milliseconds) for an election to timeout. See [Documentation/tuning.md](https://etcd.io/docs/v3.4/tuning/#time-parameters) for details.
    default: “1000”
    env variable: ETCD_ELECTION_TIMEOUT